src/main/resources/anniv5Resources/localization/eng/witchesstrikepack/Keywordstrings.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "PROPER_NAME": "Manifest",
+    "NAMES": [
+      "manifest"
+    ],
+    "DESCRIPTION": "Gain an orb slot then channel the listed orb lose an orb slot when a Manifested orb is evoked."
+  },
+  {
+    "PROPER_NAME": "Crescent Moon",
+    "NAMES": [
+      "Crescent Moon",
+      "crescent moon",
+      "Crescent_Moon",
+      "crescent_moon"
+    ],
+    "DESCRIPTION": "Orb. Passive : Deals damage to a random enemy when activated. Evokes after activating twice. Evoke : Deal damage to a random enemy for each remaining activation and lose and orb slot"
+  },
+  {
+    "PROPER_NAME": "Full Moon",
+    "NAMES": [
+      "Full Moon",
+      "full moon",
+      "Full_Moon",
+      "full_moon"
+    ],
+    "DESCRIPTION": "Orb. Passive : Draw a card when activated. Evokes after activating 4 times. Evoke : Draw a card for each remaining activation and lose and orb slot"
```

Comment:
Activated is when? Passive effect triggers from cards? Or is that something specific?

Reply:
Packs orbs need to be directly activated via effects like Loop or other members of the pack.